packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -343,6 +332,9 @@ abstract class NestingTableState<T, S extends StatefulWidget> extends State<S> {
   }
 }
 
+/// Callback for when a specific item in a table is selected.
+typedef ItemCallback<T> = void Function(T item);
```

Comment:
nit: why ItemCallback instead of SelectionChangeCallback or similar.
Alternately, maybe vend a ValueListenable with the item that is selected?

Reply:
A callback is the easiest way to pass this data.

A listenable doesn't fit easily into the life cycles that are involved.  There are ways to do it, but they involve more boilerplate.